2019-05-02/greatBigG/palindrome.go

```diff
@@ -0,0 +1,53 @@
+package palindrome
+
+import (
+	"log"
+	"regexp"
+	"strings"
+)
+
+// reverse takes a string as input, reverses it
+// and returns it
+func reverse(s string) string {
+	runes := []rune(s)
```

Comment:
Good choice for using runes over iterating over string, as I understand using runes goes exactly through one character from ASCII

Reply:
Yes runes goes through one ASCII character at a time, using strings will go through one byte at a time IIRC.